app/src/components/formdemo/FormDemo.tsx

```diff
@@ -380,7 +380,7 @@ export default function FormDemo() {
 				<DxButton type="secondary" onClick={() => console.log('Secondary button clicked')}>
 					Secondary
 				</DxButton>
-				<DxButton type="link" onClick={() => console.log('Link button clicked')}>
+				<DxButton type="link" link="./" onClick={() => console.log('Link button clicked')}>
```

Comment:
Whats `link="./"` for?

Reply:
to test that the "open in a new tab" feature works, this just opens the current link in a new tab